Build failed in RedHat 8.5

teeio-validator build failed in RedHat 8.5.
gcc version is 8.5.0 20210514 (Red Hat 8.5.0-10)
```
[ 87%] Built target spdm_device_secret_lib_sample
tee-io-validator/teeio-validator/teeio_validator/ide_test_ini.c: In function ‘dump_test_config’:
tee-io-validator/teeio-validator/teeio_validator/ide_test_ini.c:2442:25: error: ‘main’ is usually a function [-Werror=main]
   IDE_TEST_MAIN_CONFIG *main = &test_config->main_config;
                         ^~~~
tee-io-validator/teeio-validator/teeio_validator/ide_test_ini.c: In function ‘dump_test_config’:
tee-io-validator/teeio-validator/teeio_validator/ide_test_ini.c:2442:25: error: ‘main’ is usually a function [-Werror=main]
   IDE_TEST_MAIN_CONFIG *main = &test_config->main_config;
                         ^~~~
make[2]: Warning: File 'tee-io-validator/teeio-validator/teeio_validator/teeio_validator.c' has modification time 41790 s in the future
```

--- a/teeio-validator/teeio_validator/ide_test_ini.c
+++ b/teeio-validator/teeio_validator/ide_test_ini.c
@@ -2439,10 +2439,10 @@ void dump_test_config(IDE_TEST_CONFIG *test_config)
 
   TEEIO_DEBUG((TEEIO_DEBUG_VERBOSE, "Dump test_ini:\n"));
 
-  IDE_TEST_MAIN_CONFIG *main = &test_config->main_config;
+  IDE_TEST_MAIN_CONFIG *main_config = &test_config->main_config;
   TEEIO_DEBUG((TEEIO_DEBUG_VERBOSE, "[Main]\n"));
-  TEEIO_DEBUG((TEEIO_DEBUG_VERBOSE, "  pci_log=%s\n", main->pci_log == 0 ? "false":"true"));
-  TEEIO_DEBUG((TEEIO_DEBUG_VERBOSE, "  debug_level=%s\n", get_ide_log_level_string(main->debug_level)));
+  TEEIO_DEBUG((TEEIO_DEBUG_VERBOSE, "  pci_log=%s\n", main_config->pci_log == 0 ? "false":"true"));
+  TEEIO_DEBUG((TEEIO_DEBUG_VERBOSE, "  debug_level=%s\n", get_ide_log_level_string(main_config->debug_level)));
   TEEIO_DEBUG((TEEIO_DEBUG_VERBOSE, "\n"));
 
   IDE_TEST_PORTS_CONFIG *ports = &test_config->ports_config;

